src/drivers/gps/gps.cpp

```diff
@@ -794,26 +813,7 @@ GPS::task_main()
 					_rate_rtcm_injection = 0.0f;
 				}
 			}
-
-			/* select next mode */
```

Comment:
This scanning should not be removed since its the best approach to search for the correct driver.